site/docs/v1/tech/identity-providers/openid-connect/azure-ad.adoc

```diff
@@ -20,7 +20,7 @@ Once logged in, navigate to [breadcrumb]#Azure Active Directory -> App Registrat
 
 image::identity-providers/azure-openid-connect-register-application.png[Register a new Azure AD Application,width=1200]
 
-Here we have configured our application `Redirect URI`.  If FusionAuth is running at `https://local.fusionauth.io`, this value should be `https://local.fusionauth.io/oauth2/callback`.
+Here we have configured our application `Redirect URI`.  If FusionAuth is running at `\https://login.fusionauth.io`, this value should be `\https://login.fusionauth.io/oauth2/callback`.
```

Comment:
@mooreds I am confused.  

Do you mean that we want this to be `login.fusionauth.io`?  We seem to be using the `login` subdomain in other tutorials which I why I updated.  Let me know if I am not fully seeing something. 🙂

Reply:
Sorry if I wasn't clear. I was suggesting changing the text to `login.piedpiper.com` because that is what is used across the other pages.

But that would require taking new screenshots (or photoshopping images) because the existing screenshots for these pages use login.fusionauth.io. 

So I was asking for your thoughts on which inconsistency was best:

* using login.fusionauth.io on this example, where other places we use login.piedpiper.com
* having the text be login.piedpiper.com but the image be login.fusionauth.io

(or, I suppose, retaking the azure screenshots is an option, not sure how much work that is.)

After some further thought, I think we should go with login.fusionauth.io (so keep the image and text consistent).

So, no action required.